2-With-broker/README.md

```diff
@@ -81,8 +81,10 @@ As a first step you'll need to:
    - In the **Supported account types** section, select **Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)**.
 1. Select **Register** to create the application.
 1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
-1. In the list of pages for the app, select **Authentication**..
+1. In the list of pages for the app, select **Authentication**.
    - In the **Redirect URIs** | **Suggested Redirect URIs for public clients (mobile, desktop)** section, check **the option of the form msal&lt;clientId&gt;://auth**
+   - If you have the new expirence, Select **Add a platform** under the **Platform Configurations** section, click on **Mobile and desktop applications** and add **msal&lt;clientId&gt;://auth** as a redirect URI.
```

Comment:
>expirence [](start = 25, length = 9)

sp: experience...also they may or may not know if they have the new experience or not

Reply:
Good point. How should we handle this @jmprieur? on my UI i see where it says preview but customers may have difficulty figuring out which experience they are seeing. Should I stick to just talking about the old experience? Do you know when the new authentication experience will be released?